device_backends/LogicalNameMapping/include/LNMAccessorPlugin.h

```diff
@@ -105,6 +105,9 @@ namespace ChimeraTK::LNMBackend {
     ~AccessorPlugin() override = default;
     friend Derived;
 
+   protected:
+    bool _needSharedTarget{false};
```

Comment:
Please add a descriptive comment. Why do we need a new member variable in the base class for all accessor plugins and what does it do?